src/include/OpenImageIO/platform.h

```diff
@@ -192,15 +189,16 @@
 #if (__cplusplus >= 202001L)
 #    define OIIO_CPLUSPLUS_VERSION 20
 #    define OIIO_CONSTEXPR20 constexpr
-#    define OIIO_INLINE_CONSTEXPR inline constexpr
 #elif (__cplusplus >= 201703L) || (defined(_MSC_VER) && _MSC_VER >= 1914)
 #    define OIIO_CPLUSPLUS_VERSION 17
 #    define OIIO_CONSTEXPR20 /* not constexpr before C++20 */
-#    define OIIO_INLINE_CONSTEXPR inline constexpr
 #else
 #    error "This version of OIIO is meant to work only with C++17 and above"
 #endif
 
+// DEPRECATED(3.1): use C++17 inline constexpr
+#define OIIO_INLINE_CONSTEXPR inline constexpr
```

Comment:
Do we really need to keep this macro ? From what I can see you've updated all the `OIIO_INLINE_CONSTEXPR` in the code base, so maybe we can just get rid of it, this way we have one less macro :)

Reply:
It's in a public header file, we can't break downstream source compatibility, even if it's something we don't use internally any longer.

Reply:
Ah you're right, good point !